pkg/kubelet/apis/BUILD

```diff
@@ -8,7 +8,6 @@ load(
 go_library(
     name = "go_default_library",
     srcs = [
-        "well_known_annotations.go",
         "well_known_annotations_windows.go",
```

Comment:
Where do you imagine the eventual home of `well_known_annotations_windows.go` being? It feels a little strange to have `well_known_annotations.go` in one package and `well_known_annotations_windows.go` in another?

Reply:
In this case I think it's fine leaving it as is. `well_known_annotatons.go` is being removed here because we are moving the only annotation in that file since it belongs somewhere else. As long as `well_known_annotations_windows.go` contains kubelet specific annotations it seems fine to leave it.

Reply:
:+1: I don't feel super strongly :)

Out of curiosity, would there be any downside to moving `well_known_annotations_windows.go`? Is the issue that we'd be moving Kubelet specific annotations into a more generic portion of the code base?

Reply:
it's strange to have OS-specific API constant values... API consumers using those annotation keys should be able to build/use those constants in clients running on other operating systems